### DISCUSSION TITLE
Simple application using webauthn4j

### DISCUSSION THREAD

**Discussion by santhoshit8** [webauthn4j/webauthn4j#690] [Q&A] [answered]
Hi all,

is there any simple application using webauthn4j like simple github project.

Thanks 
Santhosh

**Comment by ynojima** [comment, marked as answer]
This is not a direct sample application, but webauthn4j-spring-security has a simple sample app: https://github.com/webauthn4j/webauthn4j-spring-security/tree/master/samples/mpa
